facebook-instant-articles.php

```diff
@@ -439,4 +448,52 @@ function invalidate_all_posts_transformation_info_cache( $option ) {
 	}
 	add_action( 'updated_option', 'invalidate_all_posts_transformation_info_cache', 10, 1 );
 
+	function fbia_indicator_column_heading( $columns ) {
+		$publishing_settings = Instant_Articles_Option_Publishing::get_option_decoded();
+		$display_warning_column = $publishing_settings[ 'display_warning_column' ];
+
+		if( "1" === $display_warning_column ) {
+			$columns['FBIA'] = "<span title='Facebook Instant Article Distribution Status' class='fbia-col-heading'>FBIA Status</span>";
+		}
+		return $columns;
+	}
+	add_filter('manage_posts_columns', 'fbia_indicator_column_heading');
+
+	function fbia_indication_column( $column_name, $post_ID ) {
+		$publishing_settings = Instant_Articles_Option_Publishing::get_option_decoded();
+		$display_warning_column = $publishing_settings[ 'display_warning_column' ];
+
+		if( "1" === $display_warning_column ) {
+			$red_light = "<span title='Instant article is empty after transformation.' class='instant-articles-col-status error.'></span>";
```

Comment:
In these html strings, the double quotes should be part of the html and the single quotes in the code, unless i'm missing something... so for this one:
$red_light = '<span title="Instant article is empty after transformation." class="instant-articles-col-status error."></span>';